src/select/select.js

```diff
@@ -209,6 +210,7 @@ class Select extends React.Component<PropsT, StatelessStateT> {
           overrides={{
             Root: Root,
             Input: Input,
+            InputContainer: StyledInputContainer,
```

Comment:
This is putting a border around the drop-down opener (what do we call that?) component now; intended?

![image](https://user-images.githubusercontent.com/2263443/45444919-b47c1c80-b67d-11e8-8674-92a8adf69d62.png)

Reply:
it was always a part of original Input component. It's very root one, which allows to arrange `input` field and other pieces of it in flexbox model. DropDown is outside of it

![image](https://user-images.githubusercontent.com/1851469/45446256-446f9580-b681-11e8-802c-2e0620b4d009.png)

Reply:
But it's not part of the design for Select, right? Can we disable it?

Reply:
it is a part of Select styles still. That's why I used it and had some styles changed here

Reply:
I'm slightly confused, I don't see a border in the design. Maybe it's just a matter of passing in extra props here to disable the border?

![image](https://user-images.githubusercontent.com/2263443/45522564-9d6e2500-b778-11e8-87a2-32fa5c529ff4.png)

Reply:
its present in Search mode, you look at Select mode, which's fix is not in master yet
![image](https://user-images.githubusercontent.com/1851469/45522817-4c5f3080-b77a-11e8-8c69-0d684803dd0f.png)

Reply:
@abmai oh, I see your point. Yes, it's not in design. Let me remove it.